conf/bk_server.conf

```diff
@@ -956,3 +956,53 @@ storage.serve.readonly.tables=false
 
 # the cluster controller schedule interval, in milliseconds. default is 30 seconds.
 storage.cluster.controller.schedule.interval.ms=30000
+
+
+#############################################################################
+## Netty Allocator Settings
+#############################################################################
+
+# Define the memory pooling policy.
+# Available options are:
+#   - PooledDirect: Use Direct memory for all buffers and pool the memory.
+#                   Direct memory will avoid the overhead of JVM GC and most
+#                   memory copies when reading and writing to socket channel.
+#                   Pooling will add memory space overhead due to the fact that
+#                   there will be fragmentation in the allocator and that threads
+#                   will keep a portion of memory as thread-local to avoid
+#                   contention when possible.
+#   - UnpooledHeap: Allocate memory from JVM heap without any pooling.
+#                   This option has the least overhead in terms of memory usage
+#                   since the memory will be automatically reclaimed by the
+#                   JVM GC but might impose a performance penalty at high
+#                   throughput.
+# Default is: PooledDirect
+# allocatorPoolingPolicy=PooledDirect
+  
+# Controls the amount of concurrency for the memory pool.
+# Default is to have a number of allocator arenas equals to 2 * CPUS.
```

Comment:
on a 48 core machine, that's 96 pools. Seems a bit high.

Reply:
That's current Netty default